package.json

```diff
@@ -6,7 +6,7 @@
     "export-tests": true
   },
   "scripts": {
-    "pretest": "test -z \"$JENKINS_HOME\" || (echo | node bin/setup-mysql.js)",
```

Comment:
The `echo |` is still needed because `node bin/setup-mysql.js` is prompting for a password.